@here/harp-mapview/lib/Tile.ts

```diff
@@ -682,18 +682,28 @@ export class Tile implements CachedResource {
 
     /**
      * @internal
-     * Sets the tile's ground elevation range.
-     *
-     * @param minElevation Minimum tile elevation in meters.
-     * @param maxElevation Maximum tile elevation in meters.
+     * Gets the tile's ground elevation range in meters.
      */
-    setElevation(minElevation: number, maxElevation: number) {
-        if (minElevation === this.m_minElevation && maxElevation === this.m_maxElevation) {
+    get elevationRange(): ElevationRange {
+        return this.m_elevationRange;
+    }
+
+    /**
+     * @internal
+     * Sets the tile's ground elevation range in meters.
+     *
+     * @param elevationRange The elevation range.
+     */
+    set elevationRange(elevationRange: ElevationRange) {
+        if (
+            elevationRange.minElevation === this.m_elevationRange.minElevation &&
+            elevationRange.maxElevation === this.m_elevationRange.maxElevation &&
+            elevationRange.calculationStatus === this.m_elevationRange.calculationStatus
+        ) {
             return;
         }
 
-        this.m_minElevation = minElevation;
-        this.m_maxElevation = maxElevation;
+        this.m_elevationRange = elevationRange;
```

Comment:
Would be safer to make a copy of this object, elevationRange object probably (from caller) may changed out of this function scope.

Reply:
True. I'm gonna let the caller do the copy though if it's needed. I'll change it in VisibleTileSet.

Reply:
I think it is good to do copy here, because the caller does not always knows what is the reference passed used for... This way single responsibility chain remains on the user (accessor class) to make sure object is not modified.
BTW we should already have some standards grown on such stuff... Good to ask on the team's channel.

Reply:
ok.